scripts/exportOpenApi.gradle

```diff
@@ -14,38 +14,54 @@ buildscript {
 }
 apply plugin: 'org.openapi.generator'
 
-def specFile = config.openApi.specFile
+
+// For a list of supported options see https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator-gradle-plugin
+def specFile = findProperty("openApi.specFile")?:config.openApi.specFile
 
 //tag::exportOpenApi[]
 task exportOpenApi (
         type: org.openapitools.generator.gradle.plugin.tasks.GenerateTask,
         group: 'docToolchain',
         description: 'exports OpenAPI specification to the asciidoc file') {
 
+    doFirst {
+        if (!specFile) {
+            throw new GradleException("Missing property 'openApi.specFile': please provide the location of the OpenAPI specification file")
+        }
+
+        logger.debug("\n=====================\nProject Config:\n=====================")
+        logger.debug("Docdir: ${docDir}")
+        logger.debug("Target: ${targetDir}")
+        logger.info("\n=====================\nOpenAPI Config:\n=====================")
+        logger.info("Specification file: ${specFile}")
+        logger.info("inputSpec: ${inputSpec}")
+        logger.info("outputDir: ${outputDir}\n")
+    }
+
     if (!specFile) {
-        logger.info("\n---> OpenAPI specification file not found in Config.groovy (https://doctoolchain.github.io/docToolchain/#_exportopenapi)")
+        // If the specFile is not defined skip the rest. Otherwise GenerateTask
+        // is called which fails with a misleading error message "file doesn't exist".
+        // We throw the descriptive "missing property" error message during execution phase.
         return
-    } else {
-        logger.info("Found OpenAPI specification in Config.groovy")
     }
```

Comment:
This also looks strange. But I had to add this `return` statement otherwise `GenerateTask` would throw an exception and I never reached the code in doFirst() which contains a better error message.

Reply:
need to investigate a bit, on the first glance it's because you use just a 3rd [partie task](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator-gradle-plugin/src/main/kotlin/org/openapitools/generator/gradle/plugin/tasks/GenerateTask.kt) and the implementation is just offering this behaviour. Maybe there is something to turn off the validation